packages/gql/src/balancer-gauges/gauge.ts

```diff
@@ -7,3 +7,34 @@ export const gaugeSymbol = gql`
     }
   }
 `;
+
+export const poolsWithGauge = gql`
+  query PreferentialGauge {
+    pools(first: 1000, where: { gauges_: { gauge_not: null } }) {
+      poolId
+      gauges {
+        id
+      }
+    }
+  }
+`;
+
+export const gaugeInfo = gql`
+  query PoolPreferentialGauge($poolId: Bytes!) {
+    pools(where: { poolId_in: [$poolId], gauges_: { gauge_not: null } }) {
+      id
+      poolId
+      preferentialGauge {
+        id
+      }
+      gauges {
+        gauge {
+          liquidityGauge {
+            id
+            symbol
+          }
+        }
+      }
```

Comment:
```suggestion
      gauges {
            id
            symbol
      }
```

Reply:
feel free to ignore if it causes much trouble down the line, but `gauges` already has `id` and `symbol` and you can use it